sdk/core/src/dto/shared.dto.ts

```diff
@@ -207,11 +206,11 @@ export class SignedCredential {
 
   @IsOptional()
   @IsISO8601()
-  issuanceDate: string
+  issuanceDate?: string
```

Comment:
(a question)
Some of these fields are required by the W3C spec, but why are they optional here?
Can signed credential be incomplete?